docs/deployment.md

```diff
@@ -33,6 +33,33 @@ Consult the documentation on each service for more information on how they can b
 
 * Audit (@libero/audit)
 
+### Journal login
+
+TODO: move this somewhere else
+
+For journal authentication, a configuration setting needs to be set to ensure the redirect returns the user to the correct url after authentication. The following
+describe how to change this setting in each environment. Note that builder-configuration changes requires admin approval (@diversimix)
+
+#### Staging (journal environment: continuumtest):
+URL: https://continuumtest--cdn-journal.elifesciences.org/
+
+1. : Change https://github.com/elifesciences/builder-configuration/blob/master/pillar/environment-continuumtest-public.sls#L58
```

Comment:
...to what value?

Reply:
We should also mention the review process here ... as not everyone can approve changes in builder.

Reply:
i think it does say that...

Reply:
https://github.com/libero/reviewer/pull/500#discussion_r350268461